README.md

```diff
@@ -1,2 +1,17 @@
-# Hello-microverse
-linters 
+# Project Name
+Hello-microverse
+# Description the project.
+This project that deals with linters, setting up my working space.
+# Built With
+HTML
+CSS
+# Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.